docs/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb.md

```diff
@@ -29,10 +29,10 @@
 
      * [Ubuntu 1604](https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/${release_version}/nebula-${release_version}.ubuntu1804.amd64.deb)
 
-     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 1.0.0-rc2 Centos 7 的安装包，那么可以直接通过命令下载。
+     * 链接中 `${release_version}` 为具体的发布版本号，例如要下载 `1.0.0-rc4` `Centos 7.5` 的安装包，那么可以直接通过命令下载。
 
      ```bash
-     $ wget https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/1.0.0-rc2/nebula-1.0.0-rc2.el7-5.x86_64.rpm
+     $ wget https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/package/1.0.0-rc4/nebula-1.0.0-rc4.el7-5.x86_64.rpm
```

Comment:
The oss download domain name in the Chinese document can be changed to oss-cdn.nebula-graph.com.cn
The oss download domain name in the English document can be changed to oss-cdn.nebula-graph.io

Reply:
@Amber1990Zhang can you help me to submit a new pr for this url